epam-assets/theme/theme_loveship.scss

```diff
@@ -674,21 +651,30 @@
     /* FlexRow */
     --uui-flex-row-border-bottom: 1px solid var(--uui-divider);
 
-    /* modification of the states of table components */
+    /* DataTable vars */
+    --uui-dt-border: var(--gray30);
 
-    :global(.uui-dt-cell-border) {
-        --uui-dt-cell-border: var(--gray30);
-    }
+    --uui-dt-cell-bg: transparent;
+    --uui-dt-cell-resize-bg: #EF55331F;
+    --uui-dt-cell-editor-bg-invalid: var(--red-lightest);
```

Comment:
may be we will remove 'editor' word here, just leave 'uui-dt-cell-bg-invalid'. Ando also wo all cases below.

Reply:
I do not agree, because the variables refer to the other component